cluster/gce/gci/configure-helper.sh

```diff
@@ -758,9 +758,55 @@ contexts:
 EOF
   fi
 
-if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
-    # This is the config file for the image review webhook.
-    cat <<EOF >/etc/gcp_image_review.config
+  if [[ ${WEBHOOK_GKE_EXEC_AUTH:-} ]]; then
+    if [[ -z ${EXEC_AUTH_PLUGIN_URL:-} ]]; then
+      echo "You requested GKE exec auth support for webhooks, but EXEC_AUTH_PLUGIN_URL was not specified.  This configuration depends on gke-exec-auth-plugin for authenticating to the webhook endpoint."
+      exit 1
+    fi
+
+    if [[ -z ${TOKEN_URL:-} || -z ${TOKEN_BODY:-} || -z ${TOKEN_BODY_UNQUOTED:-} ]]; then
+      echo "You requested GKE exec auth support for webhooks, but TOKEN_URL, TOKEN_BODY, and TOKEN_BODY_UNQUOTED were not provided.  gke-exec-auth-plugin requires these values for its configuration."
```

Comment:
These should be `1>&2 echo "...."`, but this file isn't consistent.

Reply:
Done